### DISCUSSION TITLE
Benesse pocket challenge W

### DISCUSSION THREAD

**Discussion by Stanton54** [sanni/cartreader#863] [General Chat]
How can I emulate this rom? I have looked online and haven't found anything.

The CRC32 checked out on one so I am curious if the others I have work.

It is a PCW file type.

Thanks!

![image](https://github.com/sanni/cartreader/assets/120297611/ae385aec-5d75-44ad-bc5f-30d0beaf880c)

**Comment by PsyK0p4T** [comment]
Glad to see someone else with interest in this little console.
I've to find again the URL but someone on no-intro managed to develop an emulator for it.

On a side note, PCW should receive a big update in the next few days. Keep an eye on the github ;)

**Reply by Stanton54** [reply]
Thanks so much!

I'll keep an eye open

**Reply by Stanton54** [reply]
![image](https://github.com/sanni/cartreader/assets/120297611/8a3e22ec-435f-4845-856b-5d174bdc81ec)

In case anyone is curious I used kindred (emulator) for windows.

The name of a lot of dumps on no intro for the system was overload who gave links to the emulator on a nes dev forum.

Thanks for the nudge in the right direction!

**Reply by PsyK0p4T** [reply]
PCW.ino and PCW.txt were both updated. Don't hesitate to update your reader and submit your redumps to Datomatic, any help with this little console will be really welcome :)
